src/main/scala/uk/gov/hmrc/domain/Random.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.domain
+
+class Random(random: scala.util.Random = new scala.util.Random) {
+  def this(seed: Int) = this(new scala.util.Random(seed))
+
+  private val invalidFirstPrefixCharacters = List("D", "F", "I", "Q", "U", "V")
```

Comment:
It feels a bit wrong to have Nino#invalidPrefixes and Random#invalidFirstPrefixCharacters, can they be unified somehow?